odoo/addons/base/models/ir_ui_view.py

```diff
@@ -2363,8 +2360,10 @@ def reset_view_button(self):
         return {'type': 'ir.actions.act_window_close'}
 
 
-class Model(models.AbstractModel):
-    _inherit = 'base'
+class Base(models.AbstractModel):
+    _name = "base"
```

Comment:
Why is the `_name` needed here?

Reply:
No, it's remove in the fix. because the upgrade script does not change the base model